pkg/chartverifier/checks/checks.go

```diff
@@ -53,6 +53,13 @@ const (
 	ImageCertified               = "Image is Red Hat certified"
 	ImageNotCertified            = "Image is not Red Hat certified"
 	ChartTestingSuccess          = "Chart tests have passed"
+	MetadataFailure              = "Empty metadata in chart"
+	RequiredAnnotationsSuccess   = "No missing required annotations"
+	RequiredAnnotationsFailure   = "Missing required annotations"
+)
+
+var (
+	requiredAnnotations = [...]string{"charts.openshift.io/name"}
```

Comment:
I would expect the certification workflow to set the correct name for index.yaml.  In the annotation we reqiure in chart.yaml we should be more general.  Maybe displayname or similar.

Reply:
Good point, not sure why we did it that way but it looks like it was me :-).